Source/Resource/Plugin/Language/fr.csv

```diff
@@ -934,293 +934,293 @@ Language_wo;Wolof
 Language_xh;Xhosa
 Language_yi;Yiddish
 Language_yo;Yoruba
-Language_yue;
+Language_yue;Cantonais
 Language_za;Zhuang
 Language_zh;Chinois
-Language_zh-cmn;
+Language_zh-cmn;Mandarin
 Language_zh-cn;Chinois (Chine)
-Language_zh-CN;
-Language_zh-Hans;
-Language_zh-Hant;
+Language_zh-CN;Chinois (Chine)
+Language_zh-Hans;Chinois (Simplifié)
+Language_zh-Hant;Chinois (Traditionnel)
 Language_zh-tw;Chinois (Taiwan)
-Language_zh-TW;
-Language_zh-yue;
+Language_zh-TW;Chinois (Taiwan)
+Language_zh-yue;Cantonais
 Language_zu;Zoulou
 LawRating;Classé
 LCCN;LCCN
-LfeAttenuationKnown;
-LfeMixGain;
-Library;Bibliotheque utilisée
+LfeAttenuationKnown;Atténuation basse fréquence connue
+LfeMixGain;Gain de mixage basse fréquence
+Library;Bibliothèques utilisées
 Lifetime_Subscribe_Button;S'abonner à vie pour %PRICE%
 Lightness;Luminosité
-Lines_Count;
-Lines_MaxCountPerEvent;
-LineUpStart;
-LinkedTo_Bed_Pos;
-LinkedTo_ChannelFormat_Pos;
-LinkedTo_ComplementaryObject_Pos;
-LinkedTo_Content_Pos;
-LinkedTo_Group_Pos;
-LinkedTo_Object_Pos;
-LinkedTo_PackFormat_Pos;
-LinkedTo_SignalGroup_Pos;
-LinkedTo_StreamFormat_Pos;
-LinkedTo_Substream_Pos;
-LinkedTo_TrackFormat_Pos;
-LinkedTo_TrackUID_Pos;
+Lines_Count;Nombre de lignes
+Lines_MaxCountPerEvent;Nombre maximum de lignes par événement
+LineUpStart;Début de l'alignement
+LinkedTo_Bed_Pos;Lits #s

```

Comment:
I would keep all the LinkedTo_ lines as in English as it is names from a spec, without translation. So copy/paste from en.csv

Reply:
I actually think that it is better not to translate the English name of the language because everyone knows the English name of their own language